### DISCUSSION TITLE
Need Crontab feature in ACAP

### DISCUSSION THREAD

**Discussion by rafiomeon** [AxisCommunications/discussions#109] [ACAP] [answered]
Hi,

Our ACAP application has to execute a script at particular time of day. We can use crontab in Linux where we could not find cron feature within Axis camera. Could you please let me know how to execute a script at a particular time of day within ACAP application

Thanks,
Rafi

**Comment by shadysp** [comment, marked as answer]
Unfortunately there are no function in the SDK for this purpose.

You could test with _systemd timers_, run this command in the terminal after connecting to the camera through ssh and see if it is possible on your camera to use it.
`systemctl status *timer `

Or you could use <sys/time.h> in your code or something similar.
